docs/reporting.md

```diff
@@ -43,15 +43,17 @@ even if you feel the information is incomplete. We have some tricks up our sleev
 Debug Logs
 ==========
 
-The debug logs that can be generated with SRT provide very detailed descriptions of its internal behaviour. In
-fact, they can sometimes approach the equivalent of "record and
-replay" for a testing session. Having the debug logs collected is
-in most cases essential to start researching a potential problem.
-
-As SRT is very highly time-based software, however, the
-usability of a debugger is very limited. Debug logs put a great burden on the performance, and for this reason have been
-shifted to the "heavy logging" category. They are not enabled by default,
-neither in the library itself, nor at compile time. You can only manually enable them at compile time:
+The debug logs that can be generated with SRT provide very detailed
+descriptions of its internal behaviour. In fact, they can sometimes approach
+the equivalent of "record and replay" for a testing session. Having the debug
+logs collected is in most cases essential to start researching a potential
+problem. This is because, as SRT is very highly time-based software, the
+usability of a debugger is very limited.
+
+Mind though, that debug logs put a great burden on the performance, and for
```

Comment:
"Mind" is more typically written as "Keep in mind"

Reply:
Right.